src/spec/design-token-spec/base-design-token-set.ts

```diff
@@ -4,10 +4,10 @@ import { DesignTokenSet } from "../../design-token/types";
 
 export const BaseDesignTokenSet: DesignTokenSet = {
     InputBoxShadow: css`
-        inset 0 0 6px 1px ${Color.Shadow.Accent}
+        inset 0 0 3px 0px ${Color.Shadow.Accent}
```

Comment:
in the thread you mentioned the blur property should be 4px, is this the final intended style?

Reply:
yeah, I got the 4px from Figma's CSS styling, but it didn't look the same when rendered. 
so I adjusted the values to look more similar to the components in Figma